src/Microsoft.ApplicationInsights/Managed/Shared/Extensibility/Implementation/Tracing/DiagnosticsTelemetryModule.cs

```diff
@@ -43,6 +47,53 @@ public DiagnosticsTelemetryModule()
         }
 
         /// <summary>
+        /// Gets or sets a value indicating whether or not the Health Heartbeat feature is disabled.
+        /// </summary>
+        public bool IsHeartbeatEnabled
+        {
+            get
+            {
+                return this.HeartbeatProvider.IsEnabled;
+            }
+
+            set
+            {
+                this.HeartbeatProvider.IsEnabled = value;
+            }
+        }
+
+        /// <summary>
+        /// Gets or sets the delay between heartbeats.
+        /// </summary>
+        public TimeSpan HeartbeatInterval
+        {
+            get => this.HeartbeatProvider.Interval;
+            set => this.HeartbeatProvider.Interval = value;
+        }
+
+        /// <summary>
+        /// Gets or sets property names that are not to be sent with the health heartbeats. null/empty list means allow all default properties through.
+        /// 
+        /// <remarks>
+        /// Default properties supplied by the Application Insights SDK:
+        /// - runtimeFramework
+        /// - baseSdkTargetFramework
+        /// </remarks>
+        /// </summary>
+        public IList<string> ExcludedHeartbeatProperties
+        {
+            get
+            {
+                return this.HeartbeatProvider.DisabledDefaultFields.ToList();
```

Comment:
this approach prevents from specifying excluded headers from `ApplicationInsights.config` file

Reply:
Understood. I found a better way to go about this I believe.